File: JavaScript/프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {
    
    let res = 0;
    let sq = Math.sqrt(n);
    for(let i=1; i<sq; i++)
        if(!(n % i))
            res += 2; // (1, n)(n, 1) 1쌍이므로 2씩
    
    if(!(n % sq)) // 같은 숫자의 경우
        res++;
    
    return res;
    
}

File: Java/프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.java
class Solution {
    public int solution(int n) {
        
        int answer = 0;
        for(int i = 1; i <= Math.sqrt(n); i++) {
            
            int ahrt = n / i;
            if(n % i == 0) {
                if(ahrt == i) answer += 1;
                else answer += 2;
            }
            
        }
        
        return answer;
        
    }
}